conf/defaults.ini

```diff
@@ -1972,3 +1972,7 @@ feedback_url = https://docs.google.com/forms/d/e/1FAIpQLSeEE33vhbSpR8A8S1A1ocZ1B
 # How frequently should the frontend UI poll for changes while resources are migrating
 frontend_poll_interval = 2s
 
+[frontend_dev]
+# Should UI tests fail when console log/warn/erroring?
+# Does not affect the result when running on CI - only for allowing devs to choose this behaviour locally
+fail_tests_on_console = true
```

Comment:
I think you need to copy this chunk into the sample.ini and comment it all out. 👍

Reply:
I'm unsure about this... What exactly is the sample.ini for?

My impression is that the sample.ini is copied across in the released artefact, and is for end-users to use as a reference for creating their server config? If that's the case, I don't feel it's appropriate to put this dev/build only stuff in there if it'll have zero impact on running grafana.